app/components/candidate-preamble.jsx

```diff
@@ -26,11 +27,11 @@ const styles = {
       left: '-100vw',
     },
     '& ul': {
-      paddingTop: '0.5em',
-      paddingLeft: '2.5em',
+      paddingTop: '0.5em!important',
```

Comment:
I don't recall but why do we have to use important ? 😱👻

Reply:
a class could  be used  to remove important as long as whatever selector is used is more specific than the style that overrides it